src/copentimelineio/anyDictionary.cpp

```diff
@@ -58,15 +58,15 @@ OTIO_API AnyDictionaryIterator* AnyDictionary_erase_range(
 }
 OTIO_API int AnyDictionary_erase_key(AnyDictionary* self, const char* key)
 {
-    return reinterpret_cast<OTIO_NS::AnyDictionary*>(self)->erase(key);
+    return static_cast<int>(reinterpret_cast<OTIO_NS::AnyDictionary*>(self)->erase(key));
 }
 OTIO_API int AnyDictionary_size(AnyDictionary* self)
```

Comment:
we should revisit why we're returning int here, where size_t makes more sense and is available from C

Reply:
Should that be a separate PR since it's an API change? I can remove the ```static_cast```s from this PR if we want to keep the warnings as a reminder.

Reply:
yes, you are absolutely right about that